### DISCUSSION TITLE
Config Path Priority

### DISCUSSION THREAD

**Discussion by Quyzi** [spf13/viper#1191] [Q&A] [answered]
Is there an order of priority for multiple config paths? I'm wondering if I can rely on viper to always prefer the config file found in `/etc/project` if it exists. If it finds multiple files, how does it decide which to use? 

```
	viper.AddConfigPath("/etc/project")
	viper.AddConfigPath("/etc/default/project")
	viper.AddConfigPath(".")
```

**Comment by sagikazarmark** [comment, marked as answer]
Viper traverses the list of config paths in the order they are added. If it finds a config file, it stops searching.

https://github.com/spf13/viper/blob/cf6565fd72e3c054f1ac13d7fd2ab7851c64c66d/viper.go#L2130-L2135
